conf/nginx/nginx.conf

```diff
@@ -17,9 +17,10 @@
 master_process on;
 
 worker_processes 1;
+worker_cpu_affinity auto;
```

Comment:
We should keep the original content, this file is intended primarily for developers.